docs/bigquery-cdcTarget.md

```diff
@@ -82,6 +82,12 @@ name is same as source database name. A valid name should only contain letters,
 maximum length can be 1024. Any invalid chars would be replaced with underscore in the final dataset name and
 any characters exceeds length limit will be truncated.
 
+**Allow Flexible Column Naming**:
+By default ony english letters, numbers and underscore are allowed in column names. If this option is enabled,
+international characters are also allowed in column names with some extra special characters, which follow the
```

Comment:
It would help if we can list a few chars which will get supported by this change.

Reply:
Updated docs, incuded
> Some special characters allowed in flexible column names are: 
> - An ampersand (&)
> - A percent sign (%)
> - A colon (:)
> - A lessthan sign (<)
> - A space ( )